server/src/util/logger.ts

```diff
@@ -0,0 +1,15 @@
+import winston from 'winston'
+
+const format = winston.format.combine(
+  winston.format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss:ms' }),
+  winston.format.colorize({ all: true }),
+  winston.format.printf(
+    (info) => `${info.timestamp} ${info.level}: ${info.message}`
+  )
+)
+
+export const logger = winston.createLogger({
+  level: 'info',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/c0dejohn/e-commerce-backend/pullRequest?prid=8292833)
```suggestion
  level: "info",
```